biz.aQute.bndlib/src/aQute/bnd/osgi/Constants.java

```diff
@@ -204,6 +204,12 @@
 	String				STRICT										= "-strict";
 	String				SUB											= "-sub";
 	String				REPRODUCIBLE								= "-reproducible";
+
+	String				RESOLVE										= "-resolve";
+	String				RESOLVE_MANUAL								= "manual";
+	String				RESOLVE_AUTO								= "auto";
+	String				RESOLVE_ONLAUNCH							= "onlaunch";
```

Comment:
Are these values supposed to align with `ResolutionInstructions.ResolveMode`? `onlaunch=beforelaunch`?